client/connection_test.go

```diff
@@ -17,28 +18,23 @@ func Test_CloseConnection(t *testing.T) {
 
 	client := CreateTestClient(container, t)
 
-	testEvent := messages.ProposedEvent{
-		EventID:      uuid.FromStringOrNil("38fffbc2-339e-11ea-8c7b-784f43837872"),
-		EventType:    "TestEvent",
-		ContentType:  "application/octet-stream",
-		UserMetadata: []byte{0xd, 0xe, 0xa, 0xd},
-		Data:         []byte{0xb, 0xe, 0xe, 0xf},
-	}
-	proposedEvents := []messages.ProposedEvent{
-		testEvent,
-	}
+	testEvent := messages.NewBinaryProposedEvent("TestEvent", []byte{0xb, 0xe, 0xe, 0xf}).
+		EventID(uuid.FromStringOrNil("38fffbc2-339e-11ea-8c7b-784f43837872")).
+		Metadata([]byte{0xd, 0xe, 0xa, 0xd})
 
 	streamID, _ := uuid.NewV4()
 	context, cancel := context.WithTimeout(context.Background(), time.Duration(5)*time.Second)
 	defer cancel()
-	_, err := client.AppendToStream(context, streamID.String(), stream_revision.StreamRevisionNoStream, proposedEvents)
+	opts := options.AppendToStreamOptionsDefault().ExpectedRevision(stream_revision.NoStream())
```

Comment:
```suggestion
	opts := options.AppendToStreamOptionsDefault().ExpectedRevision(streamrevision.NoStream())
```

Would rename the directory to remove the underscore `_`

Reply:
or move this from stream_revision to `stream/revision` :wink: more about that convention you can find here: https://golang.org/doc/effective_go#package-names